### PR TITLE
Chrome 62 released

### PR DESCRIPTION
**Pull request by afmenez**
Chrome 62 was released yesterday.  Updating metadata and test results.

### CODE REVIEW

File: environments.json [hunk 1]
```diff
@@ -1497,22 +1497,30 @@
     "short": "CH&nbsp;61,<br>OP&nbsp;48",
     "release": "2017-09-05",
     "note_id": "experimental-flag",
-    "obsolete": false
```
Comment (ljharb) [inline environments.json]: I'm not comfortable with v61 being marked obsolete just because v62 was released.

Comment (chicoxyzzy) [reply environments.json]: Me too

Comment (afmenez) [reply environments.json]: This is how this was done so far.  Ditto for Firefox.

Comment (afmenez) [reply environments.json]: I mean, if we want to change how things work, it should be a separate PR for all browsers, not just Chrome.

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**